project_config/lmic_project_config.h

```diff
@@ -1,6 +1,7 @@
 // project-specific definitions
 //#define CFG_eu868 1
-#define CFG_us915 1
+#define CFG_au915 1
```

Comment:
we shouldn't change this in lmic_project_config.h, I think.

Reply:
yeah i agree, i committed what i use here. i probably should have sent //#define CGG_au915 instead